src/main/java/duke/data/Storage.java

```diff
@@ -0,0 +1,77 @@
+package duke.data;
+
+import duke.task.*;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Storage {
```

Comment:
Write descriptive header comments for your Storage class, which I have also otherwise noticed that all your public classes lack descriptive header comments.